client/src/js/components/bhFindSupplier.js

```diff
@@ -114,9 +114,7 @@ function FindSupplierComponent(SupplierService) {
   */
   function readInput(event) {
     if (event.keyCode === 13) {
```

Comment:
This code was not changed in this pull request so it does not need to be changed here (commenting for future reference). 

It would be best if this constant was defined elsewhere to give the code meaning - 13 doesn't provide any context. 

For example: 
```js
const ENTER_KEY = 13;

if (event.keyCode === ENTER_KEY) { //...
```